packages/hdwallet-native-vault/src/vault.ts

```diff
@@ -179,6 +180,13 @@ export class Vault extends MapVault implements IVault {
     return this;
   }
 
+  async loadFromKeystore(stringifiedKeystore: string, password: string) {
+    const keystore = JSON.parse(stringifiedKeystore);
+    const mnemonic = await decryptFromKeystore(keystore, password);
+    this.set("#mnemonic", mnemonic);
+    return this;
+  }
```

Comment:
Looks unused, but actually adds `loadsFromKeystore` to `Vault` class for clients' consumption e.g:
https://github.com/shapeshift/web/pull/8442/files#diff-cb441bd026f8d983d9243fba41e8f70444867e130f150f1b25a22de360dd0008R51